test/extended/operators/daemon_set.go

```diff
@@ -46,19 +45,12 @@ var _ = g.Describe("[sig-arch] Managed cluster", func() {
 			e2e.Failf("unable to list daemonsets: %v", err)
 		}
 
-		// daemonsets that have an explicit exception, every entry here must have a bug associated
-		// e.g. "openshift-x/foo": "link to bug"
-		bugsAgainstBrokenDaemonSets := map[string]string{
```

Comment:
Keep all this infrastructure.  Have a comment in the empty map, and describe what the rules are for adding to it.

Reply:
Under what conditions would we add a new bug carve out, instead of just requiring the broken DaemonSet to be fixed?